src/app/app.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { AuthService } from './services/auth.service';
+
+import en from 'src/assets/i18n/en.json';
+import ua from 'src/assets/i18n/ua.json';
+import ru from 'src/assets/i18n/ru.json';
+import { TranslateService } from '@ngx-translate/core';
+import { LanguagesService } from './services/languages.service';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
```

Comment:
According to best practices, it better to use onPush strategy ( changeDetection: ChangeDetectionStrategy.OnPush) for each component. That helps you to prevent performance issue with application.